locallib.php

```diff
@@ -1406,7 +1406,9 @@ protected function get_certificate_text($issuecert, $certtext = null) {
         $a->date = $this->get_date($issuecert, $user->id);
         $a->outcome = $this->get_outcome($user->id);
         $a->certificatecode = $issuecert->code;
-
+        $a->coursestart = $this->get_course()->startdate ? date('d/m/Y', $this->get_course()->startdate): '';
```

Comment:
The date format is hardcoded,  many places  use  other  date formats then d/m/Y,   should use a locale date format, or  use a  field  in certificate configuration where an user specify the date format